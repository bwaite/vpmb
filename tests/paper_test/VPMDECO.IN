TRIMIX DIVE TO 260 FSW       !Description of dive
3                      !Number of gas mixes
.15,.45,.40                  !Fraction O2, Fraction He, Fraction N2
.36,.00,.64                  !Fraction O2, Fraction He, Fraction N2
1.0,.00,.00                  !Fraction O2, Fraction He, Fraction N2
1                      !Profile code 1 = descent
0,260,75,1                   !Starting depth, ending depth, rate, gasmix
2                      !Profile code 2 = constant depth
260,30,1               !Depth, run time at end of segment, gasmix
99                     !Profile code 99 = decompress
3                      !Number of ascent parameter changes
260,1,-30,10                 !Starting depth, gasmix, rate, step size
110,2,-30,10                 !Change depth, gasmix, rate, step size
20,3,-10,20                  !Change depth, gasmix, rate, step size
1                      !Repetitive code 1 = repetitive dive to follow
60                     !Surface interval time in minutes
TRIMIX DIVE TO 260 FSW       !Description of dive
3                      !Number of gas mixes
.15,.45,.40                  !Fraction O2, Fraction He, Fraction N2
.36,.00,.64                  !Fraction O2, Fraction He, Fraction N2
1.0,.00,.00                  !Fraction O2, Fraction He, Fraction N2
1                      !Profile code 1 = descent
0,260,75,1                   !Starting depth, ending depth, rate, gasmix
2                      !Profile code 2 = constant depth
260,30,1               !Depth, run time at end of segment, gasmix
99                     !Profile code 99 = decompress
3                      !Number of ascent parameter changes
260,1,-30,10                 !Starting depth, gasmix, rate, step size
110,2,-30,10                 !Change depth, gasmix, rate, step size
20,3,-10,20                  !Change depth, gasmix, rate, step size
0                      !Repetitive code 0 = last dive/end of file

