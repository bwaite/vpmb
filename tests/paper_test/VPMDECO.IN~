TRIMIX DIVE TO 80 MSW
3
.15,.45,.40
.36,.00,.64
1.0,.00,.00
1
0,80,23,1
2
80,30,1
99
3
80,1,-10,3
33,2,-10,3
6,3,-3,3
0
