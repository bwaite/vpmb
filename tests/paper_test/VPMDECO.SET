&Program_Settings
Units='fsw'                         !Options: fsw or msw
Altitude_Dive_Algorithm='OFF'       !Options: ON or OFF
Minimum_Deco_Stop_Time=1.0          !Options: real positive number
Critical_Radius_N2_Microns=0.8      !Adj. Range: 0.2 to 1.35 microns
Critical_Radius_He_Microns=0.7      !Adj. Range: 0.2 to 1.35 microns
Critical_Volume_Algorithm='ON'      !Options: ON or OFF
Crit_Volume_Parameter_Lambda=7500.0 !Adj. Range: 6500 to 8300 fsw-min
Gradient_Onset_of_Imperm_Atm=8.2    !Adj. Range: 5.0 to 10.0 atm
Surface_Tension_Gamma=0.0179        !Adj. Range: 0.015 to 0.065 N/m
Skin_Compression_GammaC=0.257       !Adj. Range: 0.160 to 0.290 N/m
Regeneration_Time_Constant=20160.0  !Adj. Range: 10080 to 51840 min
Pressure_Other_Gases_mmHg=102.0     !Constant value for PO2 up to 2 atm
/

