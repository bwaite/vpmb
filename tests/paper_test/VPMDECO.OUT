                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    02-16-2010 at 04:14 pm                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 260 FSW       !Description of dive                     
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
                          Gasmix # 2  0.360  0.000  0.640
                          Gasmix # 3  1.000  0.000  0.000
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (fswg)  (fswg)  (fsw/min) |  (fswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     3.5    3.5 |    1   | Descent     0.    260.      75.0   |
  2    26.5   30.0 |    1   |                                    |   260.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at  209.7 fswg
                 Deepest possible decompression stop is  200.0 fswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (fswg) (fsw/min)  Used  | (fswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  3     2.7   32.7 |    1   |  180.    -30.0          |
  4     0.3   33.0 |    1   |                         |   180      1     33
  5     0.3   33.3 |    1   |  170.    -30.0          |
  6     0.7   34.0 |    1   |                         |   170      1     34
  7     0.3   34.3 |    1   |  160.    -30.0          |
  8     1.7   36.0 |    1   |                         |   160      2     36
  9     0.3   36.3 |    1   |  150.    -30.0          |
 10     1.7   38.0 |    1   |                         |   150      2     38
 11     0.3   38.3 |    1   |  140.    -30.0          |
 12     1.7   40.0 |    1   |                         |   140      2     40
 13     0.3   40.3 |    1   |  130.    -30.0          |
 14     1.7   42.0 |    1   |                         |   130      2     42
 15     0.3   42.3 |    1   |  120.    -30.0          |
 16     3.7   46.0 |    1   |                         |   120      4     46
 17     0.3   46.3 |    1   |  110.    -30.0          |
 18     1.7   48.0 |    2   |                         |   110      2     48
 19     0.3   48.3 |    2   |  100.    -30.0          |
 20     1.7   50.0 |    2   |                         |   100      2     50
 21     0.3   50.3 |    2   |   90.    -30.0          |
 22     1.7   52.0 |    2   |                         |    90      2     52
 23     0.3   52.3 |    2   |   80.    -30.0          |
 24     2.7   55.0 |    2   |                         |    80      3     55
 25     0.3   55.3 |    2   |   70.    -30.0          |
 26     3.7   59.0 |    2   |                         |    70      4     59
 27     0.3   59.3 |    2   |   60.    -30.0          |
 28     4.7   64.0 |    2   |                         |    60      5     64
 29     0.3   64.3 |    2   |   50.    -30.0          |
 30     7.7   72.0 |    2   |                         |    50      8     72
 31     0.3   72.3 |    2   |   40.    -30.0          |
 32     9.7   82.0 |    2   |                         |    40     10     82
 33     0.3   82.3 |    2   |   30.    -30.0          |
 34    15.7   98.0 |    2   |                         |    30     16     98
 35     0.3   98.3 |    2   |   20.    -30.0          |
 36    44.7  143.0 |    3   |                         |    20     45    143
 37     2.0  145.0 |    3   |    0.    -10.0          |
 
REPETITIVE DIVE:
 
                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    02-16-2010 at 04:14 pm                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 260 FSW       !Description of dive                     
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
                          Gasmix # 2  0.360  0.000  0.640
                          Gasmix # 3  1.000  0.000  0.000
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (fswg)  (fswg)  (fsw/min) |  (fswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     3.5    3.5 |    1   | Descent     0.    260.      75.0   |
  2    26.5   30.0 |    1   |                                    |   260.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at  209.7 fswg
                 Deepest possible decompression stop is  200.0 fswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (fswg) (fsw/min)  Used  | (fswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  3     2.3   32.3 |    1   |  190.    -30.0          |
  4     0.7   33.0 |    1   |                         |   190      1     33
  5     0.3   33.3 |    1   |  180.    -30.0          |
  6     0.7   34.0 |    1   |                         |   180      1     34
  7     0.3   34.3 |    1   |  170.    -30.0          |
  8     0.7   35.0 |    1   |                         |   170      1     35
  9     0.3   35.3 |    1   |  160.    -30.0          |
 10     0.7   36.0 |    1   |                         |   160      1     36
 11     0.3   36.3 |    1   |  150.    -30.0          |
 12     1.7   38.0 |    1   |                         |   150      2     38
 13     0.3   38.3 |    1   |  140.    -30.0          |
 14     1.7   40.0 |    1   |                         |   140      2     40
 15     0.3   40.3 |    1   |  130.    -30.0          |
 16     2.7   43.0 |    1   |                         |   130      3     43
 17     0.3   43.3 |    1   |  120.    -30.0          |
 18     2.7   46.0 |    1   |                         |   120      3     46
 19     0.3   46.3 |    1   |  110.    -30.0          |
 20     1.7   48.0 |    2   |                         |   110      2     48
 21     0.3   48.3 |    2   |  100.    -30.0          |
 22     1.7   50.0 |    2   |                         |   100      2     50
 23     0.3   50.3 |    2   |   90.    -30.0          |
 24     1.7   52.0 |    2   |                         |    90      2     52
 25     0.3   52.3 |    2   |   80.    -30.0          |
 26     2.7   55.0 |    2   |                         |    80      3     55
 27     0.3   55.3 |    2   |   70.    -30.0          |
 28     4.7   60.0 |    2   |                         |    70      5     60
 29     0.3   60.3 |    2   |   60.    -30.0          |
 30     4.7   65.0 |    2   |                         |    60      5     65
 31     0.3   65.3 |    2   |   50.    -30.0          |
 32     7.7   73.0 |    2   |                         |    50      8     73
 33     0.3   73.3 |    2   |   40.    -30.0          |
 34    12.7   86.0 |    2   |                         |    40     13     86
 35     0.3   86.3 |    2   |   30.    -30.0          |
 36    21.7  108.0 |    2   |                         |    30     22    108
 37     0.3  108.3 |    2   |   20.    -30.0          |
 38    77.7  186.0 |    3   |                         |    20     78    186
 39     2.0  188.0 |    3   |    0.    -10.0          |
 
